webui/tests/.gitignore

```diff
@@ -1,3 +1,13 @@
-node_modules/
-results/
-*.log
+# Gauge - metadata dir
+.gauge
```

Comment:
Should these patterns be restricted? (e.g. `/.gauge/` to only match the `webui/tests/.gauge` directory?)